dev/README.md

```diff
@@ -0,0 +1,75 @@
+# Development setup
+
+This directory contains the docker compose file to run an isolated environment of the full stack during development.
+
+## Docker
+
+During development, you can run each service individually using `docker` or plug them together using `docker-compose`.
+
+`docker-compose` is also used by Github actions to make sure services are working well together all the time.
+
+### All services composed with hot reload
+
+From this folder run:
+
+```
+docker-compose -f docker-compose.yml -f docker-compose.dev.yml up --build
```

Comment:
Worth making a script for this? At risk of exposing the magnitude of my own laziness...

Reply:
absolutely, done